onadata/apps/api/tests/viewsets/test_briefcase_api.py

```diff
@@ -16,7 +16,6 @@
 
 
 NUM_INSTANCES = 4
-storage = get_storage_class()()
 
```

Comment:
should this be `storage = default_storage`? Right now, the default_storage import is unused